design/templates.go

```diff
@@ -0,0 +1,46 @@
+// nolint: gochecknoglobals
+package design
```

Comment:
ten `/design` adresar mi pride taky "nezaradeny", nemohlo by to byt takto?

```
/api
  /templates
   design.go
   /gen
     openapi.*
     gen.go (embedFs)
```

Mozno su tam nejake obmedzenia, ktore som nedomyslel.

Reply:
Jo jo, nad tím jsem taky přemýšlel. imho se to maximálně tak nebude líbit linteru. 🙂 Ok, zkusím. 👍  Akorát teď nevím co je `gen.go (embedFs)`. 🤔

Reply:
https://github.com/keboola/keboola-as-code/blob/151f1fcdbbaac0b05f87d83ae163abcec7125fba/api/api.go#L8-L12

Reply:
aha, rozumím.